packages/@angular/cli/commands/build.ts

```diff
@@ -31,10 +31,12 @@ export const baseBuildCommandOptions: any = [
     description: 'define the output filename cache-busting hashing mode',
     aliases: ['oh']
   },
+  { name: 'statsJson', type: Boolean, default: false },
```

Comment:
The weird CI errors are due to this:

```
kamik@T460p MINGW64 /d/sandbox/master-project (master)
$ ng build
The "statsJson" option's name of the "build" command contains a capital letter.
Error: The "statsJson" option's name of the "build" command contains a capital letter.
    at Class.Command.validateOption (D:\work\angular-cli\packages\@angular\cli\ember-cli\lib\models\command.js:238:11)
```

If you change it to `stats-json` it should be fine.